suite-native/module-accounts/src/components/AccountsScreen.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { Screen, ScreenHeader } from '@suite-native/navigation';
+import { Box, Text } from '@suite-native/atoms';
+
+import { AccountsList } from './AccountsList';
+
+export const AccountsScreen = () => (
```

Comment:
Not sure if it's the correct place but the back button on the accounts page visible on the video likely should not be there.

Reply:
Good catch! 🚀 [f52ba5f](https://github.com/trezor/trezor-suite/pull/6199/commits/f52ba5fe5eea93f558deb63e2db1c52e6ef06bad)